core/include/gnuradio-4.0/Block.hpp

```diff
@@ -353,6 +353,7 @@ class Block : public lifecycle::StateMachine<Derived> {
 
     constexpr static bool blockingIO             = std::disjunction_v<std::is_same<BlockingIO<true>, Arguments>..., std::is_same<BlockingIO<false>, Arguments>...>;
     constexpr static bool noDefaultTagForwarding = std::disjunction_v<std::is_same<NoDefaultTagForwarding, Arguments>...>;
```

Comment:
There is no strong need for a static bool definition for `noDefaultTagForwarding` and `backwardTagForwarding`. 

It would be better if these are defined via `using ...` statements. This way they do not occupy space in the final binary representation/tables and just compile away to nothing.